core/templates/pages/story-editor-page/services/story-editor-navigation.service.ts

```diff
@@ -30,28 +30,35 @@ const STORY_PREVIEW = 'story_preview';
 })
 export class StoryEditorNavigationService {
     activeTab: string = 'story_editor';
-    chapterId: string = null;
-    chapterIndex: number = null;
+    chapterId!: string;
+    // 'chapterIndex' is null when we are navigating to a chapter with its ID.
+    chapterIndex!: number | null;
```

Comment:
Is null redundant here?

Reply:
assigning `null` as an initial value here